packages/starlight/schemas/badge.ts

```diff
@@ -4,8 +4,13 @@ const badgeSchema = () =>
 	z.object({
 		variant: z.enum(['note', 'danger', 'success', 'caution', 'tip', 'default']).default('default'),
 		text: z.string(),
+		class: z.string().optional(),
```

Comment:
If we keep this prop here instead of `BadgeComponentSchema`, we would need to document it [here](https://starlight.astro.build/reference/configuration/#badgeconfig).

Reply:
Yes, it will update the corresponding docs.